cmd/hostpathplugin/main.go

```diff
@@ -19,8 +19,11 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/openshift/projected-resource-csi-driver/pkg/controller"
```

Comment:
Looks like the imports need sorted into sections here.

Reply:
thanks update pushed